lib/service/configuration_service.dart

```diff
@@ -888,6 +897,38 @@ class ConfigurationServiceImpl implements ConfigurationService {
   @override
   Future<void> setDidShowLiveWithArt(bool value) async =>
       await _preferences.setBool(keyDidShowLiveWithArt, value);
+
```

Comment:
*can be improved later

I think we should write a function that get map/object/list from `_preferences` by `key` to decrease code duplicate

Reply:
fixed